datacache/common.py

```diff
@@ -32,15 +35,28 @@ def get_data_dir(subdir=None, envkey=None):
         subdir = "datacache"
     return appdirs.user_cache_dir(subdir)
 
+
+def build_sqlite_url(collection_name, subdir=None):
+    filename = "%s.db" % collection_name
+    path = build_path(filename, subdir)
+    return "%s%s" % ("sqlite:///", path)
```

Comment:
minor nit: I think might want to drop the third `/`. It will work fine as written as long as `path` is always an absolute path, which I think here it is here, but will fail if `path` is a relative path. Having only two slashes I think would work in either case.